server/index.js

```diff
@@ -27,7 +27,9 @@ if (DEV) {
   const compiler = webpack([clientConfig, serverConfig])
   const clientCompiler = compiler.compilers[0]
 
```

Comment:
cool, u mind adding const options = { publicPath, stats: { colors: true } }

and making app.use one line again. ...trying to make the demo as easy to read as possible.